script.js

```diff
@@ -4,33 +4,211 @@ const logo = document.querySelector('.logo');
+    seeLiveLink: 'https://binhussen.github.io/portfolio/',
+    seeSourceLink: 'https://github.com/binhussen/portfolio',
+  },
+  {
+    title: 'Professional Art Printing Data',
+    technologies: ['html', 'css', 'ruby on rails'],
+    imageURL: './assets/images/content/Snapshoot-Portfolio.svg',
+    content:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    seeLiveLink: 'https://binhussen.github.io/portfolio/',
+    seeSourceLink: 'https://github.com/binhussen/portfolio',
+  },
+];
+
+const openModal = (i) => {
+  const projectTechs = projects[i].technologies.map(
+    (tec) => `<li class="tag">${tec}</li>`,
+  );
+
+  modal.classList.toggle('hidden');
+  wrapper.classList.toggle('blur');
+  overFlow.classList.toggle('scroll-off');
+
+  modal.innerHTML = `
+  <div class="modal-header">
+          <h1 class="primary-title modal-title">${projects[i].title}</h1>
+          <img src="assets/images/icons/Icon-Cancel.svg" alt="Cancel" class="cancel"/>
+        </div>
+        <ul class="tags">${projectTechs.join('')}
+        </ul>
+        <div class="works-stories">
+          <img
+            src=${projects[i].imageURL}
+            alt="Recent Works GYMFIT Multi-Post Stories image"
+            class="avatar modal-image"
+          />
+          <div class="container">
+            <p class="text-content works-content">
+              ${projects[i].description}
+            </p>
+            <div class="btns">
+              <a href=${
+  projects[i].seeLiveLink
+}><button type="button" class="btn">
+                See Live
+                <img
+                  class="btn-icon"
+                  src="./assets/images/icons/Icon-power.svg"
+                /></button></a>
+              </button>
+             <a href=${
+  projects[i].seeSourceLink
+}><button type="button" class="btn">
+                See Source
+                <img
+                  class="btn-icon"
+                  src="./assets/images/icons/github-white.svg"
+                />
+              </button></a>
+            </div>
+          </div>
+        </div>
+  `;
```

Comment:
- [ ] Good job so far your popup is displaying :+1:  But it is not working as expected :cry:  For Desktop your popup buttons need to go down as defined in the Figma template. You can see a difference when you compare it to Figma. Please use the same spacing and follow the layout as defined in the Figma template to make them similar. :smile: 

> * [ ]  kindly follow the figma version and bring down the seelive/seesource buttons
>   ![Screenshot (269)_LI](https://user-images.githubusercontent.com/65298048/180247969-ec941a17-f319-4b3b-8dc8-c7ccf6df1414.jpg)

### Your project popup
![Screenshot from 2022-07-21 16-07-13](https://user-images.githubusercontent.com/6615394/180263408-2df9a445-48a9-49db-a8b7-9965a2584f20.png)

### Figma popup
![Screenshot from 2022-07-21 16-17-17](https://user-images.githubusercontent.com/6615394/180263782-249dc1ac-29f5-4e65-a6d7-545fbb2bd0f1.png)

Reply:
Thank you @codecaiine for your feedback, I will modify it.